package/src/store/mappers/mapMessageToStorable.ts

```diff
@@ -1,16 +1,22 @@
-import type { MessageResponse } from 'stream-chat';
+import type { FormatMessageResponse, MessageResponse } from 'stream-chat';
 
 import { mapDateTimeToStorable } from './mapDateTimeToStorable';
 
 import type { TableRow } from '../types';
 
-export const mapMessageToStorable = (message: MessageResponse): TableRow<'messages'> => {
+export const mapMessageToStorable = (
+  message: MessageResponse | FormatMessageResponse,
+): TableRow<'messages'> => {
   const {
     attachments,
     cid,
     created_at,
     deleted_at,
     id,
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    latest_reactions,
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    own_reactions,
```

Comment:
Is this still needed?

Reply:
Yeah, we need to exclude it from storage